src/inspirehep-holdingpen-js/templates/inspirehep-holdingpen-js/details/details_decision_hep.html

```diff
@@ -1,7 +1,7 @@
-<div ng-if="!vm.record.metadata.inspire_categories.length && !vm.record.metadata.arxiv_categories.length">
+<div ng-if="!vm.record.metadata.inspire_categories.length && vm.record._workflow.data_type !== 'authors'">
```

Comment:
why do you remove ``` !vm.record.metadata.arxiv_categories.length```?

Reply:
Because Micha told me that the workflow has some step to derive inspire categories from arXiv ones. So if there are arXiv categories, there should always be inspire categories.